### PR TITLE
Python Examples

### PR DESCRIPTION
**Pull request by pavankumarbn**

### CODE REVIEW

**Comment by ajpmaclean** [comment]
If I approved these, they would never be displayed on the web page. You need to update Python.md for this to happen. Please be very careful updating this file. AppendFilter, ConnectvityFilter,  should go in the **Working with 3D Data** section, GaussianSplat should go in the **Surface reconstruction** section before the *Utilities* subsection. Use Cxx.md as a guide.

**Comment by pavankumarbn** [comment]
@ajpmaclean I have updated Python.md. Please review on the same.

File: src/Python.md [hunk 1]
```diff
@@ -292,6 +294,7 @@ This section includes vtkUnstructuredGrid.
 [MedicalDemo4](/Python/Medical/MedicalDemo4) | vtkFixedPointVolumeRayCastMapper | Create a volume rendering
 
 ### Surface reconstruction
+[GaussianSplat](/Python/Filtering/GaussianSplat) | vtkGaussianSplatter | Create a surface from Unorganized Points (Gaussian Splat).
```
[the remark this answers is not in the document]
Comment (pavankumarbn) [reply src/Python.md]: @ajpmaclean I have added the table definition. Please review on the same.

**Comment by ajpmaclean** [comment]
Thanks.

**Pull request merged**